### ISSUE TITLE
Dashboard shows when starting VR content

### ISSUE THREAD

**Issue by bddckr** [ValveSoftware/openvr#1326]
tl;dr: The new behavior of SteamVR 1.10.20 and later to show the dashboard on startup breaks a lot of Unity content out there.

Requests/fix suggestions:
1. Can the new behavior be adjusted to not show the dashboard when a scene app is already running? Care needs to be taken to ensure this works in the case of rapidly starting content right after SteamVR started, but also in the case of someone starting content which then implicitly starts SteamVR due to its `OpenVR.Init` call.
2. Alternatively: Can this behavior be rolled back?
3. Alternatively: Can we get some configuration for this behavior?

---

SteamVR 1.10.20 changed the dashboard behavior a bit it seems: The dashboard is now automatically shown on startup of SteamVR (at least... I have also seen it just pop up after closing the running scene app).

The reproduction steps are:
1. Start SteamVR
2. Immediately start a VR scene app.
3. See that the VR app is running and rendering already, without issues.
4. Wait a few more seconds.
5. Dashboard shows up!

Now here's the part that makes this trigger a bug: Step 4 above can actually happen before step 3 sometimes (in my testing it happens ~80% in this bug-triggering order). So rarely the dashboard opens seconds later, but most of the time it actually opens right away, before the started app actually is calling `OpenVR.Init`.

The bug triggered exists in most content out there: The Unity SteamVR plugin used in content that has not been updated to any recent version (with the input system changes) does have issues keeping its internal state for controller rendering and input handling in check. When the dashboard is up right as the plugin initializes it will get told to not render the controller render models, and it also won't get any input forwarded to the app.
More details can be found by reading these PR descriptions: https://github.com/ValveSoftware/steamvr_unity_plugin/pull/91#issue-198972364 and https://github.com/ValveSoftware/steamvr_unity_plugin/pull/90#issue-198293011. The only way to resolve the issue is to restart the application while ensuring the dashboard is currently hidden.

Affected content won't most likely not be updated by the developers in question, as they haven't done so in years. Additionally the two linked PRs sadly won't help the developers to fix their content (assuming they'd do so and their content still brings in enough revenue to allow doing so) - I've been running these workarounds in production for more than a year now and can see that there is a ~30% chance of the bug triggering still.
